src/Features/CSharp/Portable/Completion/CompletionProviders/PropertySubPatternCompletionProvider.cs

```diff
@@ -46,10 +46,10 @@ internal class PropertySubpatternCompletionProvider : CommonCompletionProvider
                 !m.IsImplicitlyDeclared);
 
             // Filter out those members that have already been typed
-            var propertySubpattern = (PropertySubpatternSyntax)token.Parent;
+            var propertySubpattern = (PropertyPatternClauseSyntax)token.Parent;
```

Comment:
nit: it might be nice to rename the variable too

Reply:
and the method `TryGetOpenBraceOrCommaInPropertySubpattern` to `TryGetOpenBraceOrCommaInPropertyPatternClause`